src/interface/components/VLIW/modal/VLIWLoadModalComponent.tsx

```diff
@@ -61,56 +58,62 @@ export const VLIWLoadModalComponent = ({
   isLoadModalOpen,
   actions,
 }: VLIWLoadModalComponentProps) => {
-  const [modalError, setModalError] = useState("");
-  const [modalVLIWError, setModalVLIWError] = useState("");
-  const [modalCode, setModalCode] = useState(DEFAULT_MODAL_CODE);
-  const [modalVLIWCode, setModalVLIWCode] = useState(DEFAULT_MODAL_VLIW_CODE);
+  const [modalError, setModalError] = useState({ general: "", vliw: "" });
+  const [modalCode, setModalCode] = useState({
+    general: DEFAULT_MODAL_CODE,
+    vliw: DEFAULT_MODAL_VLIW_CODE,
+  });
   const [t] = useTranslation();
 
   const close = () => {
     actions.toggleLoadModal(false);
   };
 
   const loadCodeFromFile = ([[fileContent]]) => {
-    setModalCode(fileContent.target.result);
+    setModalCode({ ...modalCode, general: fileContent.target.result });
   };
 
   const loadVLIWCodeFromFile = ([[fileContent]]) => {
-    setModalVLIWCode(fileContent.target.result);
+    setModalCode({ ...modalCode, vliw: fileContent.target.result });
   };
 
   const loadCode = () => {
     const code = new Code();
     const vliwCode = new VLIWCode();
 
+    let generalError = modalError.general;
+    let vliwError = modalError.vliw;
```

Comment:
```suggestion
    let { general: generalError, vliw: vliwError } = modalError;
```

You can also use dot notation instead of assignation